### PR TITLE
Extend TOF Event Time maker for analysis

### PR DESCRIPTION
**Pull request by njacazio**
- Add authors
- Add track multiplicity used for the TOF event time estimation

Align AOD maker behaviour for trackTime
- Update trackTime
- Fix call to exp times

Expand log axis in HistogramRegistry

### CODE REVIEW

**Comment by njacazio** [comment]
@noferini Here it is!

**Comment by njacazio** [comment]
For reference this is used in https://github.com/AliceO2Group/O2Physics/pull/300

**Review by noferini — DISMISSED** [review]

File: Detectors/AOD/src/AODProducerWorkflowSpec.cxx [hunk 1]
```diff
@@ -327,8 +327,11 @@ void AODProducerWorkflowDPL::fillTrackTablesPerCollision(int collisionID,
             }
             const auto& tofCl = tofClus[contributorsGID[GIndex::Source::TOF]];
             // correct the time of the track
-            extraInfoHolder.trackTime = (tofCl.getTime() - tofInt.getTOF(trackPar.getPID())) * 1e-3; // tof time in \mus, FIXME: account for time of flight to R TOF
-            extraInfoHolder.trackTimeRes = 200e-3;                                                   // FIXME: calculate actual resolution (if possible?)
+            const float massZ = o2::track::PID::getMass2Z(trackPar.getPID());
+            const float energy = sqrt((massZ * massZ) + (extraInfoHolder.tofExpMom * extraInfoHolder.tofExpMom));
```
Comment (shahor02) [inline Detectors/AOD/src/AODProducerWorkflowSpec.cxx:331]: why using m/Z and not M ?

Comment (njacazio) [reply Detectors/AOD/src/AODProducerWorkflowSpec.cxx:331]: Hi Ruben, it is to match the p/z measurement

**Comment by noferini** [comment]
Hi @shahor02,
if you agree I would merge this PR.
I need to work on this classes for calibration stuffs and I am waiting for this PR before to proceed with my changes.

File: Detectors/AOD/src/AODProducerWorkflowSpec.cxx (hunk 1 above)
Comment (shahor02) [reply Detectors/AOD/src/AODProducerWorkflowSpec.cxx:331]: ok, makes sense, thanks.

**Review by shahor02 — APPROVED** [review]

**Pull request merged**